### ISSUE TITLE
Custom memory_size for async tasks

### ISSUE THREAD

**Issue by th0th** [Miserlou/Zappa#1326]
## tl;dr
Is it possible to set a different memory size (different than `memory_size` set in `zappa_settings.json`) for async tasks? If not, is it on the roadmap? I am not sure if AWS Lambda offers such functionality.

## A little bit detail
I have custom async tasks that does some heavy tasks asynchronously. If I set the global `memory_size` accordingly, I am wasting a certain amount of that memory. If I set a low `memory_size`, then my asynchronous tasks are going to fail.

As I described in the short description of the issue, I am not sure if there is an AWS Lambda limitation, but I it would be lovely to be able to set `memory_size` as a parameter like `@task(memory_size=512)` in `zappa.async.task` decorator or in `zappa.async.run`.

**Comment by jwkvam** [comment]
I think there's a constraint of 1 Lambda Function to 1 Zappa app. If that's true, you'd need to setup another Lambda function and use boto to invoke it.

**Comment by th0th** [comment]
@jwkvam Thanks for the explanation.
